apps/backend/src/websocket/message_protocol.py

```
@@ -0,0 +1,142 @@
+"""
+WebSocket Message Protocol
+
+Defines standardized message format and types for WebSocket communication
+between the server and connected clients.
+"""
+
+from enum import Enum
+from typing import Any, Dict, Optional, List
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)